src/ResourceManager/Automation/Commands.Automation/Cmdlet/NewAzureAutomationModule.cs

```diff
@@ -39,8 +39,9 @@ public class NewAzureAutomationModule : AzureAutomationBaseCmdlet
         /// </summary>
         [Parameter(Position = 3, Mandatory = true, ValueFromPipelineByPropertyName = true,
             HelpMessage = "The url to a module zip package.")]
+        [Alias("ContentLink")]
         [ValidateNotNullOrEmpty]
-        public Uri ContentLink { get; set; }
+        public Uri ContentLinkUri { get; set; }
```

Comment:
@vrdmr you are still referencing `ContentLink` below instead of `ContentLinkUri`, which is causing compilation errors